### PR TITLE
added new callback  to notify about date range picker change

### PR DESCRIPTION
**Pull request by f0zze**

### CODE REVIEW

File: src/components/datepicker/DateRangePicker.js [hunk 1]
```diff
@@ -171,10 +169,10 @@ class DatePicker extends React.Component {
     }
 
     updateStateTo(to, toText) {
+        let open = this.state.from === null || to === null;
```
Comment (janisAiz) [inline src/components/datepicker/DateRangePicker.js]: common logic

File: src/components/datepicker/DateRangePicker.js [hunk 2]
```diff
@@ -184,10 +182,11 @@ class DatePicker extends React.Component {
     }
 
     updateStateFrom(from, fromText) {
+        let open = this.state.to === null || from === null;
```
Comment (janisAiz) [inline src/components/datepicker/DateRangePicker.js]: common logic

Comment (f0zze) [inline src/components/datepicker/DateRangePicker.js]: hm and how can i do better?

File: src/components/datepicker/DateRangePicker.js [hunk 3]
```diff
@@ -171,10 +169,10 @@ class DatePicker extends React.Component {
     }
 
     updateStateTo(to, toText) {
```
Comment (janisAiz) [inline src/components/datepicker/DateRangePicker.js]: updateStateTo and updateStateFrom is principal the same only setState differs .... which could be given as funciton for common method updateState.

File: src/components/datepicker/DateRangePicker.js [hunk 4]
```diff
@@ -68,13 +68,11 @@ class DatePicker extends React.Component {
         if (nextProps.value) {
             let from = nextProps.value.from;
             let to = nextProps.value.to;
-            let open = isEmpty(from) || isEmpty(to);
             this.setState({
                 from: this.parseDate(from),
                 fromText: from,
                 to: this.parseDate(to),
-                toText: to,
-                open
```
Comment (janisAiz) [inline src/components/datepicker/DateRangePicker.js]: controlled state looks like won't work ... only assumption

File: src/components/datepicker/DateRangePicker.js (hunk 3 above)
Comment (f0zze) [inline src/components/datepicker/DateRangePicker.js]: and how i will know what to update!? pass some key?
add swtch???

**Pull request closed without merging**